src/routes/itty-router/typescript/+page.md

```diff
@@ -176,6 +184,20 @@ type GenericTraps = {
 type IRequest = IRequestStrict & GenericTraps
 ```
 
+<a name="HasContent"></a>
+
+### `HasContent`
+This type is intended to be used in conjunction with the [`withContent`](../api/+page.md#withContent) middleware to facilitate unpacking request payloads onto the request.
```

Comment:
the link here was a best guess - not sure it will properly resolve, not sure how to test it though